templates/zerver/integrations/onyx.md

```diff
@@ -0,0 +1,27 @@
+# Zulip Onyx Integration
+
+Capture discussions from Zulip channels and streams to your Onyx knowledge
+base!
+
+{start_tabs}
+
+1.  {!create-a-generic-bot.md!}
+
+1.  Copy the bot's credentials by clicking on the **copy**
+<i class="zulip-icon zulip-icon-copy"></i> icon under the bot's name.
+
+1. In Onyx, open the **Admin Dashboard** and select the **Zulip Connector**.
+
+1. Under **Provide Credentials**, paste the zuliprc content that you
+   copied above, and click **Update**.
```

Comment:
Huh? Nothing above says "zuliprc". For users to be able to follow the instructions, the terminology needs to be consistent throughout the steps.